Template10 (Library)/Common/Bootstrapper/BootStrapper.cs

```diff
@@ -71,34 +71,46 @@ public enum AdditionalKinds { Primary, Toast, SecondaryTile, Other, JumpListItem
         public static new BootStrapper Current => Application.Current as BootStrapper;
 
         /// <summary>
-        /// This memory-only dictionary gives developers a place to store complex types while 
+        /// This memory-only dictionary gives developers a place to store complex types while
         /// navigating, including non-serializable types that cannot be passed as parameters.
         /// </summary>
         public IStateItems SessionState { get; set; } = new StateItems();
 
         /// <summary>
         /// This property tells Template 10 if should automatically restore the NavitgationState
-        /// of Frames when the application is restored from suspension. 
+        /// of Frames when the application is restored from suspension.
         /// </summary>
         public bool AutoRestoreAfterTerminated { get; set; } = true;
 
         /// <summary>
-        /// This setting tells Template 10 if it should automatically implement a SavingData 
+        /// This setting tells Template 10 if it should automatically implement a SavingData
         /// ExtendedSession when Suspending. This extends the time limit for Suspension activity.
         /// </summary>
         public bool AutoExtendExecutionSession { get; set; } = true;
 
         /// <summary>
         /// This setting tells Template 10 if it should automatically save the NavigationState
-        /// of every NavigationService's Frame. This enables it to be restored on Resume. 
+        /// of every NavigationService's Frame. This enables it to be restored on Resume.
         /// </summary>
         public bool AutoSuspendAllFrames { get; set; } = true;
 
         /// <summary>
         /// There are many ways for an app to start and re-start, including activation.
-        /// This property retains the original (first-launch) activation arguments. 
+        /// This field retains the original (first-launch) activation arguments.
         /// </summary>
-        public IActivatedEventArgs OriginalActivatedArgs { get; private set; }
+        private IActivatedEventArgs _originalActivatedArgs;
```

Comment:
The _originalActivatedArgs is useless after my changes. It is only used to check if the current activation is the first one. If can be replaced by a bool value.